### ISSUE TITLE
Subgraph failed to start, but does not get marked as failed

### ISSUE THREAD

**Issue by lutter** [graphprotocol/graph-node#839]
Adam from Livepeer just alerted us to the fact that the Livepeer subgraph stopped syncing. On closer inspection, there's an entry in the log of `index-node-mainnet-2` to which Livepeer is assigned saying
```
Mar 25 00:50:29.685 ERRO Subgraph instance failed to start, subgraph_id: QmfPTP8Kao6kTGRRRDyHKnsQWgDU1ytAqhezJuP3Spq86A, error: subgraph resolve error: resolve error: deadline has elapsed
```
(Note that the timestamp for the log entry if `2019-03-25T00:50:30.602480892Z`, i.e., not PDT)

There are no log messages mentioning the Livepeer subgraph after that.

**Comment by Jannis** [comment]
The not getting marked as failed was fixed in https://github.com/graphprotocol/graph-node/pull/824 but has not been rolled out to production yet.

The `deadline elapsed` error is a timeout trying to fetch the subgraph manifest and related files from our IPFS node. My theory so far has been that this only happens after deploying a new subgraph version, if files take a while to be recognized by the node. But if this happens even long after the subgraph files were deployed, then the theory doesn't hold.

Other potential causes: IPFS node simply responding to `ipfs.cat` too slowly; our rate limiting kicking in when all nodes restart after an upgrade and try to read from IPFS at the same time.

Possible solutions: increase the IPFS timeout in graph-node, relax the rate limiting in our hosted IPFS node.

**Comment by yanivtal** [comment]
Increasing timeouts was a fine short term solution but at this stage we should take the hit and build dependable solutions to these problems.

**Comment by shivgupt** [comment]
Will this issue be resolved in upcoming release? I running docker graph node image and ganache keep getting same error.
Is there a work around for it?

**Comment by Jannis** [comment]
@shivgupt This should be fixed in `v0.13.0`. Make sure you pull the latest graph-node Docker image with `docker pull graphprotocol/graph-node:latest` or `docker pull graphprotocol/graph-node:v0.13.2`.

**Comment by Jannis** [comment]
Please re-open if the issue still persists.
